pages/about.md

```diff
@@ -0,0 +1,163 @@
+---
+layout: page
+title: About Solid
+permalink: /about
+
+---
+
+# About Solid
+
+Solid is a mid-course correction for the Web by its inventor, 
+[Sir Tim Berners-Lee](/team). It realizes Tim's original vision for the 
+Web as a medium for the secure, decentralized exchange of public 
+and private data.
+
+<img src="/assets/img/timbl-cern.jpg"/>
+
+## Why fix the Web?
+
+The [first web browser](https://worldwideweb.cern.ch/) was also an editor. 
+The idea being that not only could everyone read content on the web, but 
+they could also help create it. It was 
+to be a collaborative space for everyone.
+
+However, when the first browser that popularized the web came along, called 
+Mosaic, it included multimedia and editing was taken out. It was considered 
```

Comment:
*but* editing was taken out ? Sentence structure is a bit confusing

Reply:
Will clarify in next iteration